nw/assets/themes/syntax/solarized_dark.conf

```diff
@@ -0,0 +1,25 @@
+[Main]
+name       = Solarized Dark
+author     = nullbasis
+credit     = Ethan Schoonover
+url        = https://ethanschoonover.com/solarized/
+license    = MIT
+licenseurl = https://github.com/altercation/solarized/blob/master/LICENSE
+
+[Syntax]
+background     =   7,  54,  66
+text           = 253, 246, 227
+link           =  38, 139, 210
+headertext     = 147, 161, 161
+headertag      =  42, 161, 152
+emphasis       =  38, 139, 210
+straightquotes = 238, 232, 213
+doublequotes   = 238, 232, 213
+singlequotes   = 238, 232, 213
```

Comment:
It's fine that the `singlequotes` and `doublequotes` colours are the same, but the `straightquotes` is used to indicate that a quoted piece of text has different quote symbols than those specified in the user preferences. It should therefore be a different colour, preferably some tone of red to stand out.

Reply:
So, when I turn off Auto-Replace in my preferences, straight quotes still get the red color specified for them in the theme. Should I open an issue for this? When straight quotes are what I'm after I don't think they should stand out like something is wrong.

Reply:
You can turn off highlighting of quotes in the Editor section of Preferences. There's a separate switch for it. The highlighting is not connected to the auto-replace feature, although the colour is defined by the symbols chosen.

If you do want dialogue highlighting, but don't want the red colour, just set both double quote symbols in the auto-replace tab to straight quotes, and disable the auto-replace switch.